alchemist-cognitive-agents/src/test/kotlin/it/unibo/alchemist/test/TestPedestriansLoading.kt

```diff
@@ -33,10 +37,8 @@ class TestPedestriansLoading<T, P, A> : StringSpec({
 
     "groups of pedestrians loading" {
         loadYamlSimulation<T, P>("groups.yml").startSimulation(
-            onceInitialized = { e ->
-                e.nodes.filterIsInstance<Pedestrian<T, P, A>>().forEach {
-                    println("${it.id} -> ${it.membershipGroup}")
-                }
+            onceInitialized = { environment ->
+                assert(environment.nodes.all(isSocialNode))
```

Comment:
```suggestion
                require(environment.nodes.all(isSocialNode))
```

When I say assertion, I mean a Kotest assertion...